pkg/core/parse_instruction.go

```diff
@@ -16,12 +16,22 @@ func ParseInstruction(line string, scriptDir string) (*Instruction, error) {
 
 	tokens, err := parsers.TokenizeLine(line)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to tokenize line: %w", err)
 	}
 	if len(tokens) == 0 {
 		return nil, fmt.Errorf("empty instruction")
 	}
 
+	// Convert opcode and destination to string if they are PropertyRef
```

Comment:
We shouldn't get PropertyRef here anymore with the new parser, so these lines can be removed.